public/push/push.html

```diff
@@ -1,3 +1,35 @@
 <div class="push">
-  <p>push works!</p>
+  <label class="stamp">
+    <img id="img" />
+    <input
+      type="file"
+      id="picker"
+    />
+  </label>
+  <form>
+    <div>
+      <label>タイトル</label>
+      <input
+        type="text"
+        name="title"
+        placeholder="京都旅行１日目"
```

Comment:
スタンプのタイトルが「京都旅行1日目」はちょっとデカい気がします。（スタンプ帳のタイトルと一瞬勘違いしたので:pray:）

```suggestion
        placeholder="jig.jp の社屋"
```